robots:
  a_torobo:
    default_gripper: a_torobo_gripper
  b_torobo:
    default_gripper: b_torobo_gripper

grippers:
  a_torobo_gripper:
    type:         GenericGripper
    action_ns:    /torobo/a_torobo_gripper_controller/gripper_cmd
    min_position: 0.0
    max_position: 0.1
    max_effort:   5.0
  b_torobo_gripper:
    type:         GenericGripper
    action_ns:    /torobo/b_torobo_gripper_controller/gripper_cmd
    min_position: 0.0
    max_position: 0.1
    max_effort:   5.0

cameras:
  b_phoxi_m_camera: PhoXiCamera
